test/jasmine/tests/click_test.js

```diff
@@ -2,6 +2,7 @@ var Plotly = require('@lib/index');
 var Lib = require('@src/lib');
 var Drawing = require('@src/components/drawing');
 var DBLCLICKDELAY = require('@src/constants/interactions').DBLCLICKDELAY;
+var HOVERMINTIME = 50;
```

Comment:
Let's require [this](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/constants.js#L57) value instead of hard-coding it.